services/minespace-web/src/components/dashboard/mine/permits/Permits.tsx

```diff
@@ -31,10 +31,7 @@ export const Permits: FC<PermitsProps> = ({ mine, permits, explosivesPermits, ..
 
   useEffect(() => {
     if (!isLoaded) {
-      Promise.all([
-        props.fetchPermits(mine.mine_guid),
-        props.fetchExplosivesPermits(mine.mine_guid),
-      ]).then(() => {
+      Promise.all([props.fetchExplosivesPermits(mine.mine_guid)]).then(() => {
```

Comment:
does this component use explosives permits? I am confused why it's being passed permits as a prop and then fetching explosives permits. I would also like to take out the match params in favour of the useParams hook. I might be able to give it a little cleanup

Reply:
I tried to move as little as possible, if we want this new container component to load everything, we can do that. 

I need to load the permits before I know what tabs to show, so that's why i had to move that up.